pull_into_place/big_jobs/pip_build.py

```diff
@@ -32,6 +32,9 @@
             'fast=' + ('yes' if test_run else 'no'),
             'loop_start=' + str(workspace.loop_boundaries[0]),
             'loop_end=' + str(workspace.loop_boundaries[1]),
+            'outputs_folder=' + workspace.output_dir + "/sequence_profiles",
+            'design_number=' + '{0}_{1:06d}'.format(job_id,task_id),
```

Comment:
I'd rather not have these files in the output directory, since they aren't outputs that the user cares about.  I think it'd be better to make a new `scratch` directory in every "big job" directory, and to put these files in there.

Reply:
Also, you should add a `workspace.seqprof_dir` property (or whatever, doesn't need to be called that) to the `Workspace` class and avoid constructing this path here.